chapters/en/slides/module3/module3_06.md

```diff
@@ -0,0 +1,103 @@
+---
+type: slides
+title: 'module3_06'
+---
+
+# Pulling changes from a remote repository
+
+---
+# Pulling changes from the remote GitHub repository to synchronize your local repository
+
+<center>
+
+<img src='/module3/vc-pull.png' width="55%" alt="404 image"/>
+
+<center/>
+
+Notes: To obtain the new changes from the remote repository on GitHub, you will need to pull those changes to your own local repository. By pulling changes, you synchronize your local repository to what is present on GitHub. Additionally, until you pull changes from the remote repository, you will not be able to push any more changes yourself (though you will still be able to work and make commits in your own local repository)
+
+
+---
+
+#  Pulling changes from GitHub using JupyterLab 
+
+
+<center>
+
+<img src='/module3/vc-pull-jupyter.png' width="70%" alt="404 image"/>
+
+<center/>
+
+
+Notes: **1. The GitHub interface indicates the name of the last person to push a commit to the remote repository, a preview of the associated commit message, the unique commit identifier, and how long ago the commit was snapshotted.**
+
+---
+
+#  Pulling changes from GitHub using JupyterLab 
+
+
+<center>
+
+<img src='/module3/vc-pull-jupyter-1.png' width="70%" alt="404 image"/>
+
+<center/>
+
+
+Notes: **2. The Jupyter Git extension clone button.**
+We will now walk through how to use the Jupyter Git extension tool to pull changes to our `eda.ipynb` analysis file that were made by a collaborator.
+You can tell Git to “pull” by clicking on the cloud icon with the down arrow in JupyterLab
+
+---
+
+#  Pulling changes from GitHub using JupyterLab
+
+<center>
+
+<img src='/module3/vc-pull-jupyter-3.png' width="70%" alt="404 image"/>
+
+<center/>
+
+Notes: **3. The prompt after changes have been successfully pulled from a remote repository.**
+
+---
+#  Pulling changes from GitHub using JupyterLab
+
+<center>
+
+<img src='/module3/vc-pull-jupyter-4.png' width="70%" alt="404 image"/>
+
+<center/>
+
+Notes: **4. Changes made by the collaborator to `eda.ipynb` (code highlighted by red arrows).**
+
+Once the files are successfully pulled from GitHub, you need to click “Dismiss” to keep working.
+And then when you open (or refresh) the files whose changes you just pulled, you should be able to see them
+
+---
+# JupyterLab: Pulling changes from GitHub
+
+<center>
+
+<img src='/module3/vc-pull-jupyter-4.png' width="70%" alt="404 image" />
```

Comment:
side by side with `git log --oneline`?

And then an extra slide with `git show <hash>` + the how it looks when clicking a hash in the the UI?

Reply:
`git log` is chapter 4
`git show` could be discussed. I will write about these mini decisions in my notes.